packages/console/src/components/Dropdown/use-position.ts

```diff
@@ -0,0 +1,43 @@
+import { RefObject, useCallback, useLayoutEffect, useState } from 'react';
+
+type Position = {
+  left: number;
+  top: number;
+  width: number;
+};
+
+const safePadding = 20;
+const overlayPadding = 4;
```

Comment:
nit: better add some comments to explain what are `safePadding` and `overlayPadding`

Reply:
updated